CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 5.1.0 (2022-02-23)
+
+- Updated the vertical tab component to have valid aria roles, states, and properties. [#203](https://github.com/blackbaud/skyux-tabs/pull/203)
```

Comment:
```suggestion
- Updated the vertical tab component to include valid ARIA roles, states, and properties. [#203](https://github.com/blackbaud/skyux-tabs/pull/203)
```
Is this description correct? PR 203's description is about the accordion pattern, and I only see two instances of `aria` in the changes.